1-the-basics/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>ESTEFANIA SORA</title>
+</head>
+<body>
+    <h1>My name is Estefania Sora</h1>
+    <h2>ESTEFANIA SORA</h2>
+    <h3>Hobbies</h3>
+    <ul>
+        <li>Read</li>
+        <li>Watch movies<li>
+        <li>Eat pizza<li>
+    </ul>
+    <p>I'm an <strong>System Engineer </strong> student</p>
+    <p>My cat is <strong>very</strong>grumy</p>
+    <img height="400" width="500" src="https://s3.amazonaws.com/rtvc-assets-senalcolombia.gov.co/s3fs-public/field/image/mi-vecino-totoro-mei-satsuke-lluvia-parada-bus.jpg"alt="My new image ">
+    <br>
```

Comment:
Using `<br>` it's a bad practice because it's not a semantic tag. We are going to see how to put a line break.